liblocation/src/main/java/com/mapbox/android/core/location/LocationEngine.java

```diff
@@ -16,7 +16,7 @@
  */
 public abstract class LocationEngine {
   public enum Type {
-    GOOGLE_PLAY_SERVICES, ANDROID, REPLAY
+    GOOGLE_PLAY_SERVICES, ANDROID, MOCK, REPLAY
```

Comment:
What's the benefit of adding a redundant enum? If we have a choice i'd not encourage changes on api signatures that are unnecessary.